system_baseline/models.py

```diff
@@ -23,6 +23,7 @@ class SystemBaseline(db.Model):
 
     id = db.Column(UUID(as_uuid=True), primary_key=True, default=uuid.uuid4)
     account = db.Column(db.String(10), nullable=False)
+    org_id = db.Column(db.String(36))
```

Comment:
On the previous line, can account still be nullable=False now that some systems will in theory move to sending only org_id, or are we expecting account will still be guaranteed present for a while?  Are we planning a subsequent PR where org_id will be set to nullable=False once all are sending it?